build.gradle

```diff
@@ -112,4 +113,12 @@ test {
         events "passed", "skipped", "failed"
         exceptionFormat "full"
     }
+    finalizedBy jacocoTestReport
```

Comment:
Shouldn't we need to have `classDumpDir` here for the jacoco plugin? Same [like](https://github.com/opensearch-project/opensearch-build/blob/main/build.gradle#L97)

Reply:
I will take look on that

Reply:
`classDumpDir`can be useful for debugging purpose or in case of dynamically created classes for example when scripting engines are used. This plugin is not necessary for generating code coverage report or pushing code coverage report to Codecov.io. We can add this plugin in next PR if we really need it.

Reply:
Sounds good.